cluster-scope/base/user.openshift.io/groups/octo-ushift-dev/group.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+    name: octo-ushift-dev
+users:
+  - fzdarsky
+  - rcook
+  - rnoriega
+  - screeley
```

We use GitHub usernames here. I think it should be like this:

```suggestion
  - fzdarsky
  - cooktheryan
  - oglok
  - screeley44
```